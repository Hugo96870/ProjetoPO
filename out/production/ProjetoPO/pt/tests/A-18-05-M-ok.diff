*** tests/A-18-05-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-18-05-M-ok.outhyp	2020-12-03 15:05:39.188725900 +0000
***************
*** 28,41 ****
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: 0|R2|8B1256|3|174|174|2
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: 1|R2|3O3|4|32|32|4
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
--- 28,43 ----
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: BOOK|8B1256|P1|58|2|5|Os Lusíadas|Luís de Camões|1234567890
! BOOK|8B1256|P1|58|2|5|Os Lusíadas|Luís de Camões|12345678900|R2|8B1256|3|174|0.0|2|NÃO|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: BOX|3O3|P2|8|20|100|EXPRESS
! BOX|3O3|P2|8|20|100|EXPRESS1|R2|3O3|4|32|0.0|4|NÃO|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
***************
*** 61,67 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|206|0
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
--- 63,69 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
***************
*** 112,118 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|206|32
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
--- 114,120 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
***************
*** 163,169 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|SELECTION|206|206
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
--- 165,171 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
***************
*** 171,177 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: R2|Jorge Figueiredo|Lisboa, Portugal|SELECTION|206|206
  S2|Ricardo Figueiredo|Lisboa|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
--- 173,179 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0
  S2|Ricardo Figueiredo|Lisboa|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
