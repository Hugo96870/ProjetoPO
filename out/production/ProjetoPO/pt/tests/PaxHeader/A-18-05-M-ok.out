30 mtime=1606841711.281574634
